tests/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
+        "behat/behat": "^3.0",
+        "drupal/drupal-extension": "~3.0"
+    },
+  "config": {
+    "bin-dir": "bin/"
+  },
+    "require": {
+        "jarnaiz/behat-junit-formatter": "^1.3",
```

Comment:
If you change the behat version to "~3.1" this junit formatter extension is no longer required. All you need it to run behat with "behat -f junit -o xml" and copy the "tests/failures" if any to $CIRCLE_TEST_REPORTS and $CIRCLE_ARTIFACTS